tsdb/README.md

```diff
@@ -11,3 +11,9 @@ Based on the Gorilla TSDB [white papers](http://www.vldb.org/pvldb/vol8/p1816-te
 Video: [Storing 16 Bytes at Scale](https://youtu.be/b_pEevMAC3I) from [PromCon 2017](https://promcon.io/2017-munich/).
 
 See also the [format documentation](docs/format/README.md).
+
+A series of blog posts explaining different components of TSDB:
```

Comment:
Should we put a version number in here so it'll be obvious when this is outdated?

Reply:
The dates in the blog post should be enough I believe (same with the above references which will go outdated before this, we don't have a version number there too)

Reply:
> which will go outdated before this

I mean some of them :P

Reply:
Should we add them to the storage.md in the docs also / instead?

Reply:
I would take it as "also", because the main references to understand TSDB is in this README file.
I don't have strong opinions on adding it in storage.md, maybe we could reference this readme itself there?

Reply:
Or move all of this to storage.md and reference the website here instead.

Reply:
This is all developer docs, it shouldn't go on the main user-facing website.

Reply:
Alright, I will undo the last change